### DISCUSSION TITLE
Changing the way sliding works?

### DISCUSSION THREAD

**Discussion by poperigby** [redeclipse/base#1180] [Help and Support]
In my opinion, having to jump into a slide is quite awkward, and limits the ease of performing a slide. I think it would be a good idea to just be able to tap the crouch key while running to slide.

**Comment by no-lex** [comment]
The reason I have decided to stay in favor of requiring jump into slide is that the player is forced to be vulnerable (midair) in order to get the slide speed boost. This is a balance concern, not a convenience one.

**Comment by poperigby** [comment]
How do other games with sliding balance this?

**Comment by IceflowRE** [comment]
in 1.6 you can double tap W to slide. 
edit: + crouch

**Comment by SniperGoth** [comment]
> in 1.6 you can double tap W to slide.

That's a dash, not a slide though, both are different, but you can chain dash-to-slide, which i did a lot without bothering with jumping first, was more intuitive.

As for 2.0, we don't have the dash anymore so..

**Comment by IceflowRE** [comment]
no you can crouch and double tap W, which leads immediatly into an slide

**Comment by SteveHeist** [comment]
Jump-into-slide is the same way Q4 & QC (Slash) handles it... It's just a practice thing not so much an unwieldy thing.

**Comment by qreeves** [comment]
Comparing to other games is a pointless endeavour anyway, because of one simple fact: _Red Eclipse isn't those other games and makes no attempt to be like them_. We balance according to the needs of our game, not to toe the line with others.

We made a conscious decision to remove and/or prevent the "jerky" overly-high-speed aspects of our movement system in v2.0 to focus on providing a more accessible and balanced environment for everyone to enjoy, not just the "pros". Forcing someone into the air to slide slows them down momentarily enough to offer a chance to attack them before they speed off again. It is purposefully "not easy".

I get that our decisions are often unpopular with a vocal minority, especially recently with those used to v1.6, but I can't keep entertaining uninformed opinions from every random person. We've spent years agonising over every variable, testing and iterating on it to get where we are. It's like reading the blurb on the back of a book and thinking you know the entire story. I don't want to be rude, I've just had enough of it.

**Comment by poperigby** [comment]
I thought the world repository was supposed to be for discussion. I'm not telling you how to run your game, I'm just starting a discussion based on something I noticed.

**Comment by qreeves** [comment]
I get that, which is why I explained my position instead of ripping you a new one. You need to realise that the phrasing matters. Instead of saying _"in my opinion it should be like this"_, it's better to start with _"is there a reason why it is the way it is?"_. Understanding the subject matter is the key to constructive feedback.

**Comment by poperigby** [comment]
I get it. I was trying to make it clear by saying "in my opinion" and putting a question mark in my title, but I guess I need to be more explicit.

**Comment by poperigby** [comment]
To me, it would feel more intuitive if there was a sliding animation. I think that's my main problem with it. So the gameplay wouldn't have to be changed.

**Comment by SteveHeist** [comment]
To point again at QC (yeah, "we aren't the same" but I need an example), when you slide, the first-person viewmodel turns visibly. It's a useful cue to whether your slide was successful and when it ends.

**Comment by no-lex** [comment]
https://github.com/redeclipse/actors/blob/master/player/powerslide_forward.iqm

**Comment by qreeves** [comment]
I understand, really the only visual feedback you get is from your legs, so unless you're looking down instead of where the action is you probably won't notice it. I've got a couple of ideas on ways of possibly improving that.

**Comment by poperigby** [comment]
I think your legs coming out in front of you would be good. This is assumming you're doing something like a slide-tackle, instead of sliding on your needs.

**Comment by qreeves** [comment]
They already do, it's just not usually in your field of vision.

**Comment by poperigby** [comment]
I see. Maybe move the camera down lower so you can actually see?

**Comment by no-lex** [comment]
Ok, how would rifle2 scope make any sense then? The reality of human PoV means that you don't usually see your own legs.

**Comment by poperigby** [comment]
I meant only move the camera down while sliding.

**Comment by no-lex** [comment]
It does get moved down.

**Comment by qreeves** [comment]
I've already got alternative ideas jotted down, leave it with me, it needs proper experimentation on my end.

**Comment by poperigby** [comment]
Move it down more, then?

**Comment by poperigby** [comment]
> I've already got alternative ideas jotted down, leave it with me, it needs proper experimentation on my end.

Alright.
